react-accordion-component/my-app/src/Accordion.js

```diff
@@ -0,0 +1,5 @@
+import RenderTopics from './RenderTopics';
+
+export default function Accordion(props) {
+  return <RenderTopics topics={props.topics} />;
```

Comment:
I don't understand why you have a separate RenderTopics component. Why not just put all that code here, in Accordion?